pkg/generator/data/main.go.tpl

```diff
@@ -13,7 +16,7 @@ func main() {
 	{{- end}}
 	body := strings.NewReader(data.Encode())
 	{{- else}}
-	body := bytes.NewBufferString("{{.Request.Body.String}}")
+	body := bytes.NewBufferString("{{.Request.Body.String | safeString}}")
```

Comment:
The syntax I feel can work is: ```body := bytes.NewBufferString(`{{.Request.Body.String | safeString}}`)```

Reply:
Yes, it solves the problem of escaping

Reply:
This PR is in WIP status and I will continue to improve it.